example/gates_zoo.jl

```diff
@@ -0,0 +1,74 @@
+# # A non-complete list of quantum gates
+using Yao, Yao.ConstGate, YaoSym
```

Comment:
why we need YaoSym here?

Reply:
This is an example that should the definitions of all primitive gates.

Reply:
Symbolic printing is required.